Dockerfile

```diff
@@ -33,22 +30,25 @@ ENV FOREMAN_APIPIE_LANGS=en
 ENV BUNDLER_SKIPPED_GROUPS="test development openid libvirt journald facter console"
 
 RUN \
-  microdnf install -y redhat-rpm-config git \
+  dnf install -y redhat-rpm-config git-core \
     gcc-c++ make bzip2 gettext tar \
     libxml2-devel libcurl-devel ruby-devel \
     postgresql-devel && \
-  microdnf clean all
+  dnf clean all
 
 ENV DATABASE_URL=nulldb://nohost
 
 ARG HOME=/home/foreman
 USER 1001
 WORKDIR $HOME
 COPY --chown=1001:0 . ${HOME}/
-# Adding missing gems, for tzdata see https://bugzilla.redhat.com/show_bug.cgi?id=1611117
-RUN echo gem '"tzinfo-data"' > bundler.d/container.rb
-RUN bundle install --without "${BUNDLER_SKIPPED_GROUPS}" \
-    --binstubs --clean --path vendor --jobs=5 --retry=3 && \
+RUN bundle config set --local without "${BUNDLER_SKIPPED_GROUPS}" && \
+  bundle config set --local clean true && \
+  bundle config set --local path vendor && \
+  bundle config set --local jobs 5 && \
+  bundle config set --local retry 3
```

Comment:
I wonder whether those should be `bundle config` calls vs environment variables, but this is probably not too relevant for the outcome of this PR